envs/monkey_zoo/blackbox/tests/basic_test.py

```diff
@@ -62,10 +62,8 @@ def log_failure(self, timer):
                                                                                              timer.get_time_taken()))
 
     def all_analyzers_pass(self):
-        for analyzer in self.analyzers:
-            if not analyzer.analyze_test_results():
-                return False
-        return True
+        analyzers_results = [analyzer.analyze_test_results() for analyzer in self.analyzers]
```

Comment:
Nice

Reply:
I'd rather NOT save time and get all the results from all the Analyzers - completeness over efficiency